### ISSUE TITLE
About the definition of the loss function

### ISSUE THREAD

**Issue by hust-lidelong** [xinntao/ESRGAN#4]
Hi！In the code, I don't see the definition of the loss function and the use of RelativisticGAN.Is this right?
Thanks！

**Comment by xinntao** [comment]
@hust-lidelong 
Only testing code is provided in this repo. Training codes including loss function and RelativisticGAN are provided in [BasicSR](https://github.com/xinntao/BasicSR).
